tests/sentry/api/endpoints/test_organization_config_integrations.py

```diff
@@ -21,3 +21,14 @@ def test_simple(self):
         provider = provider[0]
         assert provider["name"] == "Example"
         assert provider["setupDialog"]["url"]
+
+
+class OrganizationConfigIntegrationsProviderKeyExistsTest(APITestCase):
+    def test_simple(self):
+        self.login_as(user=self.user)
+        org = self.create_organization(owner=self.user, name="baz")
+        path = u"/api/0/organizations/{}/config/integrations/?provider_key=example".format(org.slug)
+        response = self.client.get(path, format="json")
+
+        assert response.status_code == 200, response.content
+        assert len(response.data["providers"]) == 1
```

Comment:
@NisanthanNanthakumar Let's validate we get the provider we asked for instead of just a random one.